src/views/v2/month/top-bar/datepicker.php

```diff
@@ -11,19 +11,19 @@
  *
  * @version TBD
  *
- * @var string $grid_date The current calendar grid date in the `Y-m-d` format.
- * @var string $formatted_grid_date The current calendar grid date in the format specified by the "Month and year
+ * @var string    $grid_date           The current calendar grid date in the `Y-m-d` format.
+ * @var string    $formatted_grid_date The current calendar grid date in the format specified by the "Month and year
  *                                  format" option.
+ * @var \DateTime $the_date            The Month current date object.
  */
 ?>
 <div class="tribe-events-c-top-bar__datepicker">
 	<button
 		class="tribe-common-h3 tribe-common-h--alt tribe-events-c-top-bar__datepicker-button"
 		data-js="tribe-events-top-bar-datepicker-button"
 	>
-		<?php // @todo @fe the datepicker width seems to depend on the width of the grid title, it should not. ?>
 		<time
-			datetime="<?php echo esc_attr( $grid_date ); ?>"
+			datetime="<?php echo esc_attr( $the_date->format('Y-m') ); ?>"
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 9bc3ddb8e9e195f6fe26d8e4e0f37c22</sub>